validator/sawtooth_validator/execution/scheduler_parallel.py

```diff
@@ -877,10 +877,6 @@ def set_transaction_execution_result(
 
             self._condition.notify_all()
 
-    def _unscheduled_transactions(self):
-
-        return self._txns_available.copy()
```

Comment:
Isn't this copy here thanks to possible concurrent modification of this list?

Reply:
Is the list being modified by multiple functions at once? If so, shouldn't a lock be used? I was thinking it was just so that the list could be modified during iteration. This change is only here for convenience, so I can get rid of it if need be.

Reply:
As far as I can tell, `_txns_available` is only used in functions with locks (except for `_remove_subsequent_result_because_of_batch_failure` and `_reschedule_if_outstanding`, but those are themselves only called from functions with locks).

@boydjohnson, is that right?